.gitignore

```diff
@@ -1,4 +1,5 @@
 .last_update
+package-lock.json
```

Comment:
Can we keep an alphabetical order?

Reply:
i thought ordering is first file and then folder. so i added it before folder's line.
do we keep an alphabetical order without distinguish file or folder?

Reply:
i thought ordering is first file and then folder. so i added it before folder's line.
do we keep an alphabetical order without distinguish file or folder?

Reply:
acknowledgement.